### ISSUE TITLE
reporting - error after switching to wsmgrd report storage

### ISSUE THREAD

**Issue by ra-at-diladele-com** [ra-at-diladele-com/qlproxy_external#938]
2015-08-21 14:10:19,633 File system space info - 18G size, 6.7G used, 11G available
2015-08-21 14:10:19,633 Renicing the current process to lower priority 1
2015-08-21 14:10:19,674 Number of hits  is 0
2015-08-21 14:10:19,675 Number of hosts is 0
2015-08-21 14:10:19,675 Number of ips   is 0
2015-08-21 14:10:19,675 Number of users is 0
2015-08-21 14:10:19,677 Running 'everything' report
2015-08-21 14:10:19,677 Created folder /opt/qlproxy/var/reports.new/everything
2015-08-21 14:10:19,680 Calculated report periods as 2015-06-18 => 2015-06-18
2015-08-21 14:10:19,684 Meta report finished successfully. Hits: 2866, Users: 1, IPs: 1
2015-08-21 14:10:19,787 Running Day Report for 'None'
Traceback (most recent call last):
  File "/opt/qlproxy/var/console/report.py", line 186, in <module>
    main()
  File "/opt/qlproxy/var/console/report.py", line 181, in main
    run()
  File "/opt/qlproxy/var/console/report.py", line 70, in run
    p.run(gen_dir)
  File "/opt/qlproxy/var/console/report/periodic.py", line 52, in run
    t.run(d)
  File "/opt/qlproxy/var/console/report/timeframe.py", line 61, in run
    r.run(reporter_dir)
  File "/opt/qlproxy/var/console/report/days.py", line 37, in run
    self.run_unique(out_dir)
  File "/opt/qlproxy/var/console/report/days.py", line 93, in run_unique
    r.run(d)
  File "/opt/qlproxy/var/console/report/day.py", line 45, in run
    self.detail(out_dir)
  File "/opt/qlproxy/var/console/report/day.py", line 86, in detail
    record['size_rate'] = self._percent(record['size'], total_size)
  File "/opt/qlproxy/var/console/report/base.py", line 40, in _percent
    return int((float(value) / float(total)) * 100)
TypeError: float() argument must be a string or a number

**Comment by ra-at-diladele-com** [comment]
should be closed after #939 as we redo the reporting in golang

**Comment by ea-at-diladele-com** [comment]
this has been already fixed by you as far as I understood.
